res/skins/LateNight/effect_unit.xml

```diff
@@ -93,11 +93,12 @@
                   <Text>ON</Text>
                 </State>
                 <Connection>
-                  <ConfigKey persist="true"><Variable name="group"/>,enabled</ConfigKey>
+                  <ConfigKey persist="true"></ConfigKey>
                   <ButtonState>LeftButton</ButtonState>
                 </Connection>
                 <Connection>
-                  <ConfigKey persist="true"><Variable name="group"/>,enabled</ConfigKey>
+                  <!-- FIXME: Quick hack until effect chains are implemented. -->
```

Comment:
Thanks, even so I prefer every individual commit or PR to master to be complete and not half-working pending future work or design discussion. It's a good policy, since it allows releasing 2.1 at any commit from master even if pending work doesn't get done.
 
Here I am considering the case of effect chains with multiple effects. If like me, you have a variety of effect chains in your personal branch defined with multiple effects, this prevents LateNight from working with them.